### DISCUSSION TITLE
Send file Can't be download

### DISCUSSION THREAD

**Discussion by Kerwin1202** [dani-garcia/vaultwarden#2392] [Q&A] [answered]
### Deployment environment

* vaultwarden version:   2.25.1

* Install method:  docker-compose

* Clients used: web vault

* Reverse proxy and version: caddy:2

* MySQL/MariaDB or PostgreSQL version: mariadb

* Other relevant details:

### Steps to reproduce

Normal sharing file, but can't download. I have 'DOMAIN' this environment variable.

And I manually replace localhost for the current domain name,I can download the file but there is no file name and related suffixes, and the file is broken( for jpg ).

![QQ截图20220325105019](https://user-images.githubusercontent.com/16319833/160045289-23269a1e-5293-44ae-bf14-30f70c2bfe68.jpg)

### Expected behaviour

Successful download file

### Actual behaviour

The Uri redirect to localhost

**Comment by BlackDex** [comment, marked as answer]
Please check the `/admin/diagnostics` page if all is configured ok. I think the DOMAIN variable is not configured correctly.

**Comment by Kerwin1202** [comment]
@BlackDex You are right. in `/admin` the domain url is `localhost`。Thanks
